certified-connectors/SmartGlobalGovernance/apiDefinition.swagger.json

```diff
@@ -0,0 +1,918 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Smart Global Governance",
+        "description": "Smart Global Governance allows to interact with Smart Compliance Booster solution to setup new dataflows with other applications. It can be used to automate/customize many daily tasks like exchanging data, triggering alerts or sending messages to appropriate people.",
+        "version": "1.0",
+        "contact": {
+            "name": "Smart Global Governance Support Team",
+            "email": "support@smartglobal.com"
+        }
+    },
+    "host": "eu-fr-south.console.smartglobalprivacy.com",
+    "basePath": "/api/v2",
+    "schemes": [
+        "https"
+    ],
+    "consumes": ["application/json"],
+    "produces": ["application/json"],
+    "paths": {
+        "/client": {
+            "post": {
+                "tags": [
+                    "clients"
+                ],
+                "summary": "Create a Client",
+                "description": "Create a Client and its properties",
+                "operationId": "Client-create",
+                "parameters": [{
+                        "name": "body",
+                        "in": "body",
+                        "required": false,
+                        "schema": {
+                            "$ref": "#/definitions/Client"
+                        }
+                    }
+                ],
+                "responses": {
+                    "default": {
+                        "description": "default",

```

Comment:
@Philippe-Tivolle We recommend you to replace the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.